site/_docs/history.md

```diff
@@ -27,24 +27,120 @@ For a full list of releases, see
 <a href="https://github.com/apache/calcite/releases">github</a>.
 Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
-
 <!--
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.35.0">1.35.0</a> / XXXX-XX-XX
-{: #v1-35-0}
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.36.0">1.36.0</a> / XXXX-XX-XX
+{: #v1-36-0}
 
-This release comes x months after [1.34.0](#v1-34-0),
+This release comes x months after [1.35.0](#v1-35-0),
 contains contributions from x contributors, and resolves x issues.
 
 Contributors to this release:
 x,
 y (release manager),
 z.
 
+#### Breaking Changes
+{: #breaking-1-36-0}
+
+Compatibility: This release is tested on Linux, macOS, Microsoft Windows;
+using JDK/OpenJDK versions 8 to 19;
+Guava versions 16.0.1 to 31.1-jre;
+other software versions as specified in gradle.properties.
+
+#### New features
+{: #new-features-1-36-0}
+
+#### Dependency version upgrade
+{: #dependency-1-36-0}
+
+#### Bug-fixes, API changes and minor enhancements
+{: #fixes-1-36-0}
+
+#### Build and test suite
+{: #build-1-36-0}
+
+#### Web site and documentation
+{: #site-1-36-0}
+-->
+
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.35.0">1.35.0</a> / 2023-07-13
+{: #v1-35-0}
+
+This release comes 4 months after [1.34.0](#v1-34-0),
+contains contributions from 36 contributors, and resolves 139 issues.
+
+Among other new features, we have added extensive function support. For example, we added roughly 14 new functions in BigQuery library for handling array, dates, math, and string.
+Also, we added 27 new functions in Spark library for handling array, map and about 11 new functions for other database products.
+
+Among other improvements and fixes, it is worth highlighting the following improvements:
```

Comment:
"Among other ..." appears also in the paragraph above and sounds a bit repetitive. Consider skipping the sentence altogether.

Reply:
Done.